src/common/techstack/TechStacks.jsx

```diff
@@ -8,7 +8,8 @@ const TechStack = () => {
       </h2>
       <div className="list-brand-tech-stack">
         {/* <div className='md:w-[800px] 2xl:w-[1000px] w-full flex flex-wrap m-8'> */}
-        {TechStackInfo.map((Item, idx) => {
+        {
+          TechStackInfo.length&&(TechStackInfo.map((Item, idx) => {
```

Comment:
Provide some space

Reply:
@KORLA2 what issue we are running corrently which got fixed by checking array length? its an hardcoded array. we donot need check here i think.